elasticsearch-transport/lib/elasticsearch/transport/transport/http/manticore/manticore_sniffer.rb

```diff
@@ -0,0 +1,67 @@
+module Elasticsearch
+  module Transport
+    module Transport
+      module HTTP
+        class Manticore
+          # Handles node discovery ("sniffing")
+          #
+          class ManticoreSniffer < ::Elasticsearch::Transport::Transport::Sniffer
+            ES1_RE_URL  = /\[([^\/]*)?\/?([^:]*):([0-9]+)\]/
+            ES2_RE_URL  = /([^\/]*)?\/?([^:]*):([0-9]+)/
+
+            attr_reader   :transport
+            attr_accessor :timeout
+
+            # @param transport [Object] A transport instance
+            #
+            def initialize(*args)
+              @timeout = 1
+              @state_mutex = Mutex.new
+              @stopping = false
+              super
+            end
+
+            def sniff_every(interval_seconds, &block)
+              logger.info("Will sniff every #{interval_seconds} seconds for new Elasticsearch hosts!")
+              @state_mutex.synchronize do
+                return if @sniffer_thread
+                @sniffer_thread = Thread.new do
+                  last_sniff = Time.now
+                  until @state_mutex.synchronize { @stopping }
+                    sleep 1
+                    begin
+
+                      if (Time.now - last_sniff) > interval_seconds
+                        if !!block
+                          begin
+                            block.call(hosts)
+                          rescue Exception => e
+                            @logger.warn("Error invoking Elasticsearch sniffing block! [#{e.class}] #{e.message} #{e.backtrace.join("\n")}")
+                          end
+                        end
+                        last_sniff = Time.now
+                      end
+                    rescue Exception => e
+                      logger.warn("Error while sniffing Elasticsearch Nodes! [#{e.class.name}][#{e.message}][#{e.backtrace.to_a}]") if logger
```

Comment:
for consistency sake, could we change the format of either:

line 39: `@logger.warn("Error invoking Elasticsearch sniffing block! [#{e.class}] #{e.message} #{e.backtrace.join("\n")}")`
line 45: `logger.warn("Error while sniffing Elasticsearch Nodes! [#{e.class.name}][#{e.message}][#{e.backtrace.to_a}]") if logger`